library/core/src/main/java/com/google/android/exoplayer2/offline/DownloadService.java

```diff
@@ -922,7 +924,11 @@ private void update() {
       @RequirementFlags int notMetRequirements = downloadManager.getNotMetRequirements();
       Notification notification = getForegroundNotification(downloads, notMetRequirements);
       if (!notificationDisplayed) {
-        startForeground(notificationId, notification);
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) {
```

Comment:
[This](https://github.com/google/ExoPlayer/commit/cf0ccf1d45563aea9756e2f87b82eeaeb65fc6e0) appears to be the official fix that never made it from `dev-v2` into release

Is this still better for our business use case?

Reply:
I tested this out. But, it requires us to fork several subcomponents rather than one.